src/wpt/WebCryptoAPI-test.ts

```diff
@@ -0,0 +1,394 @@
+      'X448 no deriveKey usage for base key',
+      'X448 base key is not a private key',
+      'X448 public property value is a private key',
+    ],
+  },
+  'derive_bits_keys/derive_key_and_encrypt.https.any.js': {},
+  'derive_bits_keys/derive_key_and_encrypt.js': {},
+  'derive_bits_keys/derived_bits_length.https.any.js': {
+    comment: 'To be investigated',
+    expectedFailures: [
+      "HKDF derivation with 0 as 'length' parameter",
+      "PBKDF2 derivation with 0 as 'length' parameter",
+    ],
+  },
+  'derive_bits_keys/derived_bits_length.js': {},
+  'derive_bits_keys/derived_bits_length_testcases.js': {
+    comment:
+      "This is a resource file but it's not in the resources/ directory; no tests in here",
+    skipAllTests: true,
+  },
+  'derive_bits_keys/derived_bits_length_vectors.js': {},
+  'derive_bits_keys/ecdh_bits.https.any.js': {},
+  'derive_bits_keys/ecdh_bits.js': {},
+  'derive_bits_keys/ecdh_keys.https.any.js': {},
+  'derive_bits_keys/ecdh_keys.js': {},
+  'derive_bits_keys/hkdf.https.any.js': {
+    comment: 'Test file /common/subset-tests.js not found.',
+    skipAllTests: true,
+  },
+  'derive_bits_keys/hkdf.js': {},
+  'derive_bits_keys/hkdf_vectors.js': {},
+  'derive_bits_keys/pbkdf2.https.any.js': {
+    comment: 'Test file /common/subset-tests.js not found.',
+    skipAllTests: true,
+  },
+  'derive_bits_keys/pbkdf2.js': {},
+  'derive_bits_keys/pbkdf2_vectors.js': {},
+
+  'digest/digest.https.any.js': {
+    comment: 'They expect TypeError, we have NotSupportedError',
+    expectedFailures: [
+      'empty algorithm object with empty',
+      'empty algorithm object with short',
+      'empty algorithm object with medium',
+      'empty algorithm object with long',
+    ],
+  },
+
+  'encrypt_decrypt/aes.js': {},
+  'encrypt_decrypt/aes_cbc.https.any.js': {},
+  'encrypt_decrypt/aes_cbc_vectors.js': {},
+  'encrypt_decrypt/aes_ctr.https.any.js': {},
+  'encrypt_decrypt/aes_ctr_vectors.js': {},
+  'encrypt_decrypt/aes_gcm.https.any.js': {},
+  'encrypt_decrypt/aes_gcm_256_iv.https.any.js': {},
+  'encrypt_decrypt/aes_gcm_256_iv_fixtures.js': {},
+  'encrypt_decrypt/aes_gcm_96_iv_fixtures.js': {},
+  'encrypt_decrypt/aes_gcm_vectors.js': {},
+  'encrypt_decrypt/rsa.js': {},
+  'encrypt_decrypt/rsa_oaep.https.any.js': {},
+  'encrypt_decrypt/rsa_vectors.js': {},
+
+  'generateKey/failures.js': {},
+  'generateKey/failures_AES-CBC.https.any.js': {
+    comment: 'Wrong type of error returned',
+    skipAllTests: true,
+  },
+  'generateKey/failures_AES-CTR.https.any.js': {
+    comment: 'Wrong type of error returned',
+    skipAllTests: true,
+  },
+  'generateKey/failures_AES-GCM.https.any.js': {
+    comment: 'Wrong type of error returned',
+    skipAllTests: true,
+  },
+  'generateKey/failures_AES-KW.https.any.js': {
+    comment: 'Wrong type of error returned',
+    skipAllTests: true,
+  },
+  'generateKey/failures_ECDH.https.any.js': {
+    comment: 'Wrong type of error returned',
```

Comment:
Can we make these expectedFailures?

Reply:
They're also "Too many failures to list individually". I'll make sure both comments are in.

Reply:
I assumed there  was a fatal error which caused us to add skipFailures... Makes sense.